src/ui/components/Card/index.js

```diff
@@ -8,12 +8,26 @@ export default class Card extends React.Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
-    footer: PropTypes.node,
+    footerLink: PropTypes.node,
+    footerText: PropTypes.node,
     header: PropTypes.node,
   }
 
   render() {
-    const { children, className, footer, header } = this.props;
+    const { children, className, footerText, footerLink, header } = this.props;
+
+    let footer;
+    let footerClass;
+    if (footerText && footerLink) {
+      throw new Error(
+        'You cannot specify footerLink and footerText at the same time');
```

Comment:
fwiw I would have done `footer` and `footerLink` without this error.

It's fine for this PR but I'd actually imagine supplying a link would wrap the footer in the `<Link>` (and because HTML5 lets us wrap pretty much anything in an `<a>` that'd be fine). I would have supplied just the URL too. Essentially I'd think the API would be:

```jsx
<Card footer={<div>Content</div>} footerLink=`/addon/${addon.slug}` />
// Footer content => <a href="/addon/my-addon/" class="Card-footer-link"><div>Content</div></a>
```

That said, it's for a theoretical use case in the future when we'd wrap any content.